add-topic-repos

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+
+GITHUB_API=${GITHUB_API:-https://api.github.com}
+DEFAULT_ORGANIZATION=$1
+TOPIC=$2
+exclude_pattern="(protected|private)"
```

Comment:
I didn't understand this fixed exclude pattern.

Reply:
this is a regex sample, but I could remove

Reply:
No no, it's good, leave it.. I think you could do receive it as a parameter. ❤️

Reply:
Like this 3093e96?

Reply:
yes, :)